internal/config/projectconfig/project_config.go

```diff
@@ -63,9 +53,33 @@ func (c *ZeroProjectConfig) Print() {
 	pp.Println(c)
 }
 
-func NewModule(parameters Parameters, directory string, repository string, source string) Module {
+// GetDAG returns a graph of the module names used in this project config
+func (c *ZeroProjectConfig) GetDAG() dag.AcyclicGraph {
```

Comment:
using AcyclicGraph is slick 🕶️  🕶️

Reply:
Yeah it was nice to be able to handle dependencies in what is probably going to end up being the necessary way, but also be able to do it quite easily!